examples/foc/foc_motor_b16.h

```diff
@@ -80,6 +80,9 @@ struct foc_motor_b16_s
   foc_angle_b16_t               qenco;        /* Qenco angle handler */
 #endif
   int                           foc_mode;     /* FOC mode */
+#ifdef CONFIG_EXAMPLES_FOC_HAVE_RUN
```

Comment:
Is this a new option or we already have it in Kconfig?

Reply:
Added in previous PR https://github.com/apache/incubator-nuttx-apps/pull/1010/